ginipaybank/src/doc/source/capture-features.rst

```diff
@@ -0,0 +1,280 @@
+                    }
+                    is ResultError.FileImport -> {
+                        val fileImportError = result.value as ResultError.FileImport
+                        handleFileImportError(fileImportError)
+                    }
+                }
+            }
+            CaptureResult.Empty -> {
+                handleNoExtractions()
+            }
+            CaptureResult.Cancel -> {
+                handleCancellation()
+            }
+        }
+    }
+
+    fun handleFileImportError(exception: ImportedFileValidationException) {
+        var message = ...
+        exception.validationError?.let { validationError ->
+            // Get the default message
+            message = getString(validationError.textResource)
+            // Or use custom messages
+            message = when (validationError) {
+                FileImportValidator.Error.TYPE_NOT_SUPPORTED -> ...
+                FileImportValidator.Error.SIZE_TOO_LARGE -> ...
+                FileImportValidator.Error.TOO_MANY_PDF_PAGES -> ...
+                FileImportValidator.Error.PASSWORD_PROTECTED_PDF -> ...
+                FileImportValidator.Error.TOO_MANY_DOCUMENT_PAGES -> ...
+            }
+        }
+        AlertDialog.Builder(this)
+            .setMessage(message)
+            .setPositiveButton("OK") { _, _ -> finish() }
+            .show()
+    }
+
+    fun startGiniPayBankSDKForImportedFile(importedFileIntent: Intent) {
+        // Configure capture first
+        configureCapture();
+
+        fileImportCancellationToken = 
+            GiniPayBank.startCaptureFlowForIntent(captureImportLauncher, this, importedFileIntent)
+    }
+
+Component API
+^^^^^^^^^^^^^
+
+When using the Component API you need to create a ``Document`` from the intent using
+``GiniPayBank.createDocumentForImportedFiles()`` and then continue either to the ``ReviewFragmentCompat``,
+``MultiPageReviewFragment`` or ``AnalysisFragmentCompat``:
+
+.. code-block:: java
+
+    fun startGiniPayBankSDKForImportedFile(importedFileIntent: Intent) {
+        GiniPayBank.createDocumentForImportedFiles(importedFileIntent, this, object : AsyncCallback<Document, ImportedFileValidationException> {
+            override fun onSuccess(result: Document) {
+                if (result.isReviewable) {
+                    // If you have enabled capturing documents with multiple pages:
+                    launchMultiPageReviewScreen()
+                    // OR if you didn't:
+                    launchSinglePageReviewScreen()
+                } else {
+                    launchAnalysisScreen(result)
+                }
+                finish()
+            }
+
+            override fun onError(exception: ImportedFileValidationException) {
+                handleFileImportError(exception)
+            }
+
+            override fun onCancelled() {
+                // Handle cancellation.
+            }
+        })
+    }
+
+Return Assistant
```

Comment:
I remember Tom wanted us to use a different name but I forgot which one :D Is it still a thing? Maybe we need to ask Anne.